README.md

```diff
@@ -11,58 +11,58 @@ This experimental application is published for examination and evaluation. It is
 
 ### Logon to container
 
-    docker-compose run --rm --no-deps --entrypoint /bin/bash node
+    docker-compose run --rm --entrypoint /bin/sh node
```

Comment:
Switching to `sh` because the Alpine Linux distribution (which is smaller) doesn’t ship with `bash`.